ci-scripts/dockerfiles/tutorials.dockerfile

```diff
@@ -0,0 +1,36 @@
+#
+# Execute this from the top-level ReFrame source directory
+#
+
+
+FROM reframehpc/rfm-ci-base:lmod
+
+ENV _SPACK_VER=0.16
+ENV _EB_VER=4.4.1
+
+# ReFrame user
+RUN useradd -ms /bin/bash rfmuser
+
+USER rfmuser
+
+# Install Spack
+RUN git clone https://github.com/spack/spack ~/spack && \
+    cd ~/spack && \
+    git checkout releases/v${_SPACK_VER}
+
+RUN pip3 install easybuild==${_EB_VER}
```

Comment:
```suggestion
# Install Easybuild
RUN pip3 install easybuild==${_EB_VER}
```

Reply:
@jgphpc I'd skip this comment, since the statement is self explanatory.